docs/gettingstarted/getting-started-splunk-setup.md

```diff
@@ -1,49 +1,59 @@
-consulted in the event the index specified in the event is not configured on Splunk.  Keep in mind just _one_ bad message will "taint" the
-whole batch (by default 1000 events) and prevent the entire batch from being sent to Splunk.
-* In case you are not using TLS on SC4S- turn off SSL on global settings for HEC in Splunk.
-- Refer to [Splunk Cloud](http://docs.splunk.com/Documentation/Splunk/7.3.1/Data/UsetheHTTPEventCollector#Configure_HTTP_Event_Collector_on_managed_Splunk_Cloud)
-or [Splunk Enterprise](http://dev.splunk.com/view/event-collector/SP-CAAAE6Q) for specific HEC configuration instructions based on your
+To set up syslog processing with SC4S, perform the following tasks in your Splunk instance:
+1. Create indexes within Splunk.
+2. Configure your HTTP event collector.
+3. Create a load balancing mechanism.
+
+
+## Step 1: Create indexes within Splunk
+
+SC4S maps each sourcetype to the following indexes by default:
+
+* `email`
+* `epav`
+* `epintel`
+* `infraops`
+* `netauth`
+* `netdlp`
+* `netdns`
+* `netfw`
+* `netids`
+* `netlb`
+* `netops`
+* `netwaf`
+* `netproxy`
+* `netipam`
+* `oswin`
+* `oswinsec`
+* `osnix`
+* `print`
+* `_metrics` (Optional opt-in for SC4S operational metrics; ensure this is created as a metrics index)
+
+You can also you create your own indexes in Splunk. See [Create custom indexes]( https://docs.splunk.com/Documentation/Splunk/9.2.1/Indexer/Setupmultipleindexes) for more information.
+
+## Step 2: Configure your HTTP event collector
+
+See [Use the HTTP event collector](https://docs.splunk.com/Documentation/Splunk/9.2.1/Data/UsetheHTTPEventCollector) for HEC configuration instructions based on your
 Splunk type.
 
+Keep in mind the following best practices specific to HEC for SC4S:
+* Make sure that the HEC token created for SC4S has permissions to add events to `main`, `_metrics`, and all other event destination indexes.
+* You can leave "Selected Indexes" blank on the token configuration page so that the token has access to
+all indexes, including the `lastChanceIndex`.  If you do populate this field, take extreme care to keep it up to date; an attempt to
+send data to an index that is not in this list results in a `400` error from the HEC endpoint. The `lastChanceIndex` will not be
+consulted if the index specified in the event is not configured on Splunk and the entire batch is then not sent to Splunk.
+* If you are not using TLS on SC4S, turn off SSL on global settings for HEC in Splunk.
+* SC4S traffic must be sent to HEC endpoints that are configured directly on the indexers.  
+
+### Step 2: Create a load balancing mechanism
+Create a load balancing mechanism between SC4S and Splunk indexers. See [Set up load balancing](https://docs.splunk.com/Documentation/Splunk/9.2.1/Forwarding/Setuploadbalancingd) for more information. Note that this should not be confused with load balancing between [sources and SC4S](../lb.md). 
```

Comment:
splunk docs on load balancing don't apply to this case. Can we make it more in the style of: 

```
In some situations, it is necessary to ensure balancing of the output from SC4S to Splunk indexers. Note that this should not be confused with load balancing between [sources and SC4S](../lb.md). 
```

what we mean here is that:
- if you have Splunk Cloud no worry, you're already covered and your SC4S output will be automatically load balanced to Splunk indexers
- if you have Splunk Enterprise and a single indexer, you obviously don't need an lb
- if you have Splunk Enterprise and mutliple indexers, you should load balance your SC4S output

Reply:
thanks! Is there another topic we should link to in case they are now to the product and need some guidance for creating this type of load balancing?

Reply:
Unfortunately there's nothing to link at this point, we don't provide any further recommendations for lbs at this point

Reply:
"In some configurations, you should ensure output balancing from SC4S to Splunk indexers. To do this, you create a load balancing mechanism between SC4S and Splunk indexers. See [Set up load balancing](https://docs.splunk.com/Documentation/Splunk/9.2.1/Forwarding/Setuploadbalancingd) for more information."